esp/scm/ecllib.ecm

```diff
@@ -74,11 +74,3 @@ SCMInterface IClientInstantEcl(IInterface)
 
 extern "C" ECL_CLIENT_API IClientInstantEcl * createInstantEclClient();
 
-SCMclarion
```

Comment:
As a separate task, you may want to look and see if we still need these files that use SCMInterface.  Even if we still use them it would be interesting to see how they are used and if we can move away from using them.  But again, no bearing on this PR.